CodingAChat/Scenes/Home/HomeViewController+CollectionView.swift

```diff
@@ -17,6 +17,9 @@ extension HomeViewController: UICollectionViewDataSource, UICollectionViewDelega
         header.editProfileSettings = {
             self.delegate?.onEditDetailsAction()
         }
+        header.settingsView = {
```

Comment:
use [weak self] inside a closure. (Please do it in editProfileSettings too, maybe Samuele forgot it)